packages/shopping/src/services/jwt-service.ts

```diff
@@ -54,7 +58,7 @@ export class JWTService implements TokenService {
       );
     }
     const userInfoForToken = {
-      id: userProfile[securityId],
+      id: userProfile.id,
```

Comment:
I think it should still be `userProfile[securityId]`

Reply:
userProfile.id and userProfile[securityId] have same values.

Reply:
depends on what is returned by user service 
```
  convertToUserProfile(user: User): UserProfile {
    // since first name and lastName are optional, no error is thrown if not provided
    let userName = '';
    if (user.firstName) userName = `${user.firstName}`;
    if (user.lastName)
      userName = user.firstName
        ? `${userName} ${user.lastName}`
        : `${user.lastName}`;
    return {[securityId]: user.id, name: userName, id: user.id};
  }
```

Reply:
true in this case theoretically `[securityId]` always equals to `id`, but conceptually we should retrieve the value from a user profile's unique field, which is `[securityId]`

Reply:
ok, I will correct this.